integration/tests/git_source/git_source_git.rb

```diff
@@ -5,7 +5,7 @@
 
 confine(:to, :platform => 'el')
 
-if fact_on(master, "operatingsystemmajrelease").to_i < 6
+if fact_on(master, "operatingsystemmajrelease").to_i < 6 || fact_on(master, "operatingsystemmajrelease").to_i > 8
```

Comment:
Are we should that fact is gonna be there? It's one of the ones removed in puppet 8.

Reply:
Oh I see, did it change to something else?

Reply:
`os.release.major` I think is how you'd reference it, see: https://github.com/puppetlabs/classifier/blob/main/src/puppetlabs/classifier/legacy_fact_translation.clj#L104